### ISSUE TITLE
Dead link error due to "unable to verify first certificate"

### ISSUE THREAD

**Issue by amimas** [textlint-rule/textlint-rule-no-dead-link#132]
I have a link that is being reported as dead. The error message says "unable to verify first certificate".

```
   24:25  error  https://dashboard.elasticpath.com is dead. (request to https://dashboard.elasticpath.com/ failed, reason: unable to verify the first certificate)   no-dead-link
```

It seems to be intermittent. The error doesn't happen consistently but it happens frequently. And I can't see any issues in the cert either so far.

**Comment by azu** [comment]
Its look like server problem, but https://ssltools.digicert.com/checker/views/checkInstallation.jsp say that `dashboard.elasticpath.com` certificat chain is valid.

[node.js - Error: unable to verify the first certificate in nodejs - Stack Overflow](https://stackoverflow.com/questions/31673587/error-unable-to-verify-the-first-certificate-in-nodejs)

We can add ignore option like `ignoreUnauthorized` for avoiding this issue.
